src/token.jl

```diff
@@ -118,8 +118,6 @@ function sortedcomplement(of::SourceRange, set)
 end
 
 ⨳(sym::Symbol) = Expr(sym)
-⨳(sym::Symbol, args::LineNumberNode...) = Expr(sym::Symbol, args...)
```

Comment:
It feels impossible to guard against breaking other packages that rely on unexported and untested methods but I can just add these back, no problems.